tests/main/selinux-clean/task.yaml

```diff
@@ -42,32 +42,32 @@ execute: |
     session-tool -u test test-snapd-desktop.sh -c 'echo hello world'
     session-tool -u test test-snapd-desktop.sh -c 'mkdir /home/test/foo'
     session-tool -u test test-snapd-desktop.sh -c 'echo foo >/home/test/foo/bar'
-    ausearch -i --checkpoint stamp --start checkpoint -m AVC 2>&1 | MATCH 'no matches'
+    retry-tool -n 5 --wait 2 sh -c 'ausearch -i --checkpoint stamp --start checkpoint -m AVC 2>&1 | grep -q "no matches"'
```

Comment:
We can't call ausearch with checkpoint in loop, as checkpoint is used as reference, and updated when ausearch runs. Effectively, this means that we are only checking the audit log in a range from the previous call to ausearch.

Reply:
I've opened https://github.com/snapcore/snapd/pull/8732 with an update to the policy. Runnig `spread google:fedora-32-64:tests/main/selinux...` does not raise any errors now. Once it lands, you should merge master and drop the retry-tool calls.

Reply:
@bboozzoo thanks for the change.